front/app/components/UI/ImagesDropzone/index.tsx

```diff
@@ -266,20 +262,18 @@ class ImagesDropzone extends PureComponent<Props & InjectedIntlProps, State> {
   }
 
   removeExcessImages = () => {
+    const {
+      maxNumberOfImages = ImagesDropzone.defaultProps.maxNumberOfImages,
```

Comment:
The refactoring in this component has to do with adding default props because there was a type problem with the defition of the `maxNumberOfImages` and `maxImageFileSize`.